src/meta_schedule/schedule_rule/schedule_rule.cc

```diff
@@ -85,6 +85,39 @@ Array<ScheduleRule> ScheduleRule::DefaultLLVM() {
   };
 }
 
+Array<ScheduleRule> ScheduleRule::DefaultVNNI() {
+  return {
+      ScheduleRule::AutoInline(
```

Comment:
I think you need to add `ApplyCustomRule` at the beginning, as of last week. Otherwise the fix in https://github.com/apache/tvm/pull/13346 won't be applied.

Reply:
oops @vinx13 has beaten me by 30 seconds...